forte/data/ontology/core.py

```diff
@@ -635,5 +637,214 @@ def index_key(self) -> int:
         return self.tid
 
 
+class Grid:
+    """
+    Regular grid with a grid configuration dependent on the image size.
+    It is a data structure used to retrieve grid-related objects such as grid
+    cells from the image. Grid itself doesn't store any data.
```

Comment:
A little-over claim on not storing any data, we can say `grid` only record the position/indices etc.